core/src/main/java/com/linecorp/armeria/common/logging/Sampler.java

```diff
@@ -27,7 +27,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.linecorp.armeria.server.logging;
+package com.linecorp.armeria.common.logging;
```

Comment:
We should probably put this in internal.logging to keep it out of the public API (the package could also contain those static helpers). Otherwise, it would make sense for the builder to take an arbitrary Sampler as an argument too, but probably overkill.